### PR TITLE
Fixes for Julia 1.0

### PR DESCRIPTION
**Pull request by ararslan**
Closes #6.

### CODE REVIEW

**Comment by ararslan** [comment]
Supersedes #6.

File: test/runtests.jl [hunk 1]
```diff
@@ -4,17 +4,17 @@ using Compat.Test
 using Compat.Sockets
 import Compat.Libdl
 using Compat.Distributed
+using Compat: @info
 using Base: BufferStream
-using TestSetExtensions
 
-eval(Syslogs, Meta.parse("UDP_PORT = 8080"))
-eval(Syslogs, Meta.parse("TCP_PORT = 8080"))
+Core.eval(Syslogs, Meta.parse("UDP_PORT = 8080"))
```
Comment (iamed2) [inline test/runtests.jl]: Can these just be formatted like `@eval Syslogs UDP_PORT = 8080` instead?

**Review by iamed2 — CHANGES_REQUESTED** [review]

File: test/runtests.jl (hunk 1 above)
Comment (ararslan) [reply test/runtests.jl]: Yep

File: .travis.yml [hunk 2]
```diff
@@ -5,19 +5,30 @@ os:
...
 matrix:
   fast_finish: true
   allow_failures:
     - julia: nightly
 notifications:
   email: false
-script:
-  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia -e 'Pkg.clone(pwd()); Pkg.build("Syslogs"); Pkg.test("Syslogs"; coverage=true)'
+#script:
```
Comment (rofinn) [inline .travis.yml]: Maybe just delete this?

**Review by rofinn — APPROVED** [review]
LGTM

File: .travis.yml (hunk 2 above)
Comment (ararslan) [reply .travis.yml]: Sure, can do

**Review by iamed2 — APPROVED** [review]

**Pull request merged**